pkg/apis/workspaces/v1alpha1/kubernetesLikeComponent.go

```diff
@@ -20,7 +24,7 @@ type K8sLikeComponentLocation struct {
 
 type K8sLikeComponent struct {
 	BaseComponent `json:",inline"`
-	Location K8sLikeComponentLocation `json:",inline"`
+	Location      K8sLikeComponentLocation `json:",inline"`
```

Comment:
This looks like a spurious format -- type should not align with json tag

Reply:
fixed